tutor/test_cc1_04_ExerciseEditingAndQA.py

```diff
@@ -157,7 +216,63 @@ def test_admin_add_the_content_analyst_role_to_a_user_7653(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
+        self.admin.login()
+        self.admin.open_user_menu()
+        self.admin.wait.until(
+            expect.element_to_be_clickable(
+                (By.LINK_TEXT, 'Admin')
+            )
+        ).click()
+        self.admin.page.wait_for_page_load()
+        self.admin.driver.find_element(
+            By.XPATH,
+            '//a[contains(text(),"Users")]'
+        ).click()
```

Comment:
for the scrolling, the scrolls in this function scroll to the bottom of the page, not a specific element. Should I change it to find the element it's going to at the bottom and then scroll to that, or is there a way to just scroll to the bottom of the page with the helper?